argo-cd-apps/base/host/segment-bridge/sprayproxy.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
why is this file called `sprayproxy.yaml`?

Reply:
fixed.